### DISCUSSION TITLE
About windows support

### DISCUSSION THREAD

**Discussion by NesManrique** [smallstep/certificates#392] [Q&A] [answered]
I just downloaded the latest release of step-cli executable for windows but when I try to run it, a window opens and is closed super quick, I can't even see what if I'm getting an error.

I'm testing on Windows 10 Enterprise LTSC. Is there any specific thing I have to do to run this properly on Windows?

Alternatively, is there anything I can do to check what error could I be getting?

**Comment by maraino** [comment, marked as answer]
@NesManrique are you running `step` inside a terminal console? We recommend the new [Microsoft Terminal](https://aka.ms/terminal) https://github.com/microsoft/terminal

**Reply by NesManrique** [reply]
I just donwloaded, extracted and double clicked the .exe provided in the latest release https://github.com/smallstep/cli/releases/tag/v0.15.2.

I haven't used that terminal before so I will check out if I can run the exe with it. Any pointers would help too :)

**Reply by Johannestegner** [reply]
`step-cii` is a "command line interface" application, so if you just start it from the exe it will open `cmd` and then exit right away cause there is no input.
You can open `cmd` or `terminal` and run it through there, then you should be able to use it as intended without the console closing.

It might be a good idea to put it in your `PATH` as well, but it's not needed if you are in the same folder with the terminal when starting it.

**Reply by NesManrique** [reply]
I feel dumb for not trying it with the command line before. I almost never use windows through the command line, just by running step-cli I just went through a PowerShell rabbithole.

Anyway, now it's working properly, thank so much for the help.
